packages/paste-website/src/components/site-wrapper/site-header/SiteHeaderSearch.tsx

```diff
@@ -4,13 +4,15 @@ import { SearchIcon } from "@twilio-paste/icons/esm/SearchIcon";
 import { InlineCode } from "@twilio-paste/inline-code";
 import { ScreenReaderOnly } from "@twilio-paste/screen-reader-only";
 import { Text } from "@twilio-paste/text";
+import { useWindowSize } from "@twilio-paste/utils";
 import * as React from "react";
 import { useHotkeys } from "react-hotkeys-hook";
 
 import { SiteSearch } from "../../site-search";
 
 const SiteHeaderSearch: React.FC = () => {
   const [isOpen, setIsOpen] = React.useState(false);
+  const { breakpointIndex } = useWindowSize();
```

Comment:
**question:** Do you know if we have a minimum window size we optimize for? I'm testing at 320px and it breaks, it’s almost like the background shrinks faster than the rest of the elements, or maybe it’s the scroll bar container? I was only able to test on this small screen in the browser, in my mobile device is working well. 
![Screenshot 2024-09-24 at 1 00 46 PM](https://github.com/user-attachments/assets/cc495c20-0ab3-4570-8066-5fd518af5a59)

Reply:
yeah so with current implementation, 340px is the lower limit